data/Hints/tournament.json

```
@@ -1,19 +1,23 @@
 {
     "name":                  "tournament",
     "gui_name":              "Tournament",
-    "description":           "Hint Distribution for the S5 Tournament. 5 Goal Hints, 3 Barren Hints, 5 Sometimes hints, 7 Always hints (including skull mask).",
+    "description":           "Hint Distribution for the S6 Tournament. 5 Goal hints, 7 Sometimes hints, 8 Always hints (including Skull Mask and Sheik in Kakariko).",
     "add_locations":         [
-        { "location": "Deku Theater Skull Mask", "types": ["always"] }
+        { "location": "Deku Theater Skull Mask", "types": ["always"] },
+        { "location": "Sheik in Kakariko", "types": ["always"]}
```

Let's keep the formatting consistent.

```suggestion
        { "location": "Sheik in Kakariko", "types": ["always"] }
```

---

Ooo yeah typo ty